source/Engine.cpp

```diff
@@ -2712,6 +2743,112 @@ void Engine::DoGrudge(const shared_ptr<Ship> &target, const Government *attacker
 
 
 
+size_t Engine::FleetPlacementLimit(const LimitedEvents<Fleet> &fleet, unsigned frames, bool requireGovernment)
+{
+	// frames = how many frames worth of ships to place:
+	//    0 = used to indicate the fleet.InitialCount() number of fleets should
+	//        be spawned if they aren't already present
+	//   60 = used immediately after that, five times when entering the system to
```

Comment:
I'm not tracking this...  are you saying that there are 12 frames per second?
Should these numbers be defined as named constants if they have special meanings?